tests/test_cli.py

```diff
@@ -448,12 +448,79 @@ def test_genetic_map(self):
         model = stdpopsim.PiecewiseConstantSize(species.population_size)
         engine = stdpopsim.get_default_engine()
         stdout, stderr = capture_output(
-                cli.write_citations, engine, model, contig)
+                cli.write_citations, engine, model, contig, None)
```

Comment:
i'd love it if you used a local variable here to describe the bibtex flag, rather than just pass `None`.  As written it's opaque.

Reply:
Perfect, sorry I can see now that it is definitely difficult to parse. I'll fix this while changing it over to `urllib` with @jeromekelleher's suggestions!